lispy.el

```diff
@@ -8563,14 +8563,19 @@ When ARG is non-nil, unquote the current string."
 
 (defvar lispy-mode-map-evilcp
   (let ((map (copy-keymap lispy-mode-map-base)))
-    (define-key map (kbd "M-)") 'lispy-close-round-and-newline)
+    ;; pairs
     (define-key map (kbd "[") 'lispy-open-square)
     (define-key map (kbd "]") 'lispy-close-square)
     (define-key map (kbd "{") 'lispy-open-curly)
     (define-key map (kbd "}") 'lispy-close-curly)
     (define-key map (kbd ")") 'lispy-right-nostring)
     (define-key map (kbd "\"") 'lispy-doublequote)
+    ;; wrapping
+    (define-key map (kbd "M-(") 'lispy-wrap-round)
+    (define-key map (kbd "M-{") 'lispy-wrap-braces)
+    (define-key map (kbd "M-[") 'lispy-wrap-brackets)
     (define-key map (kbd "M-\"") 'lispy-meta-doublequote)
```

Comment:
You are right. They are different and paredit-meta-doublequote is what we want.

@abo-abo Would it be okay to add something like lispy-wrap-doublequotes (or different function name) to lispy so that your package can be used without paredit?

To illustrate the function (take '|' as the cursor position)

    |this

should be turned into

    "|this"